service-front/module/Application/view/application/pages/cp/add_lpa.twig

```diff
@@ -39,88 +39,91 @@
                                 </td>
                             </tr>
                         </table>
-                        {% endif %}
-                    </div>
+                    {% endif %}
                 </div>
-            {% endif %}
+            </div>
         {% endif %}
+    {% endif %}
 
 
-            <h1 class="govuk-heading-xl">Find an LPA to add</h1>
+    <h1 class="govuk-heading-xl">Find an LPA to add</h1>
 
-            <p class="govuk-body">
-                LPA reference numbers are 12 numbers long. For
-                <br>
-                example: M-0000-0000-0000
+    <p class="govuk-body app-text-gray">
+        LPA reference numbers are 12 characters <br>
+        long. For example: M-0000-0000-0000
```

Comment:
We shouldn't be inserting individual line breaks, particularly mid-sentence. It will cause wrapping issues on certain screen sizes and add an unnecessary pause when announced by a screenreader.